src/Libraries/RevitNodes/Elements/Views/View.cs

```diff
@@ -166,15 +165,15 @@ public override string ToString()
         /// </summary>
         /// <param name="parameterFilter">Parameter filter</param>
         /// <returns name="view">View</returns>
-        public Element AddFilter(Revit.Filter.ParameterFilterElement parameterFilter)
+        public Revit.Elements.Views.View AddFilter(Revit.Filter.ParameterFilterElement parameterFilter)
```

Comment:
I understand that, but the signature of this public method changed, if you can verify that an old graph using this node before you changed the signature opens fine after your change is made we can keep this.